src/support_sphere_db/src/ssdb/models/user.py

```diff
@@ -0,0 +1,18 @@
+from typing import Optional
+
+import uuid
+from sqlmodel import SQLModel, Field, Relationship
+
+
+class User(SQLModel, table=True):
```

Comment:
Could you add docstring for this class that essentially describe it being one of the table managed by supabase in a numpy docstring format: https://numpydoc.readthedocs.io/en/latest/format.html#documenting-classes

Thanks!

Reply:
updated.